vision/product_search/product_search_add_product_to_product_set.rb

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Google, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START vision_product_search_add_product_to_product_set]
+require "google/cloud/vision"
+
```

Comment:
Remove extraneous newline

Ruby never uses 2 blank lines like Python

Reply:
(obvs remove here & in the other files, I won't spam you with bagillions of comments)